ibm/mas_devops/roles/cert_renew_check/README.md

```diff
@@ -0,0 +1,20 @@
+cert_renew_check
+=============
+
+Will run the renewal command and check that a different secret has been created so the renewal has actually taken place. It checks both public and internal certs.
+
+Role Variables
+--------------
+### mas_instance_id
+Required - Defines the instance id that is used in the existing MAS installation, will be used to lookup the existing MAS subscription and targeted Certificates, Issuers and Secrets to be migrated.
+
+- Environment Variable: `MAS_INSTANCE_ID`
+- Default Value: None
+
+### custom_cluster_issuer
+Optional - Only required if the targeted MAS 8.6 instance is configured with a custom cluster issuer (i.e Let's Encrypt).
```

Comment:
Is there something specific to MAS 8.6 here?

Reply:
Ah no - in fact I didn't use the custom cluster issuer so have removed it from the readMe